Athos/HelperScripts/process_models/change_onnx_output.py

```diff
@@ -0,0 +1,131 @@
+'''
+
+Authors: Pratik Bhatu.
+
+Copyright:
+Copyright (c) 2020 Microsoft Research
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+'''
+
+import onnx
+import onnxruntime
+import numpy as np
+from onnx import helper, shape_inference, checker
+from onnx import ValueInfoProto, ModelProto, TensorProto
+import os
+
+model_name = "shufflenet_may17.onnx"
```

Comment:
@bhatuzdaname why do we need this? We can always exchange the cleaned model between us.

Reply:
Sooo I keep receiving updated models which have different weights and a slightly different architecture. So it makes sense to have this handy then instead of manually cleaning the models as and when we receive them. I plan to use this as part of the compilation workflow for shufflenet.